packages/react-ui-kit/src/Form/Checkbox.tsx

```diff
@@ -78,7 +96,7 @@ const filterCheckboxProps = (props: CheckboxProps) => filterProps(props, ['markI
 export const Checkbox: React.FC<CheckboxProps<HTMLInputElement>> = React.forwardRef<
   HTMLInputElement,
   CheckboxProps<HTMLInputElement>
->(({id = Math.random().toString(), children, style, disabled, ...props}, ref) => (
+>(({id = useId(), children, style, disabled, ...props}, ref) => (
```

Comment:
Oh that's cool, I was not aware React had this :ok_hand:

Reply:
nice this caught my eye as well